### PR TITLE
Feature/os 67 input field floating label

### PR DESCRIPTION
**Pull request by 99999989**

### CODE REVIEW

File: online-services-ui/src/app/shared/components/input-field/input-field.component.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  <div class="form-label-group position-relative" [class.invalid]="control.touched && control.invalid">
+    <span *ngIf="required && !readonly"
+          class="required-indicator h-100">
+    </span>
+
+    <input *ngIf="!multiline && !readonly"
+           [attr.id]="id"
+           [formControl]="control"
+           [type]="type"
+           [attr.min]="min"
```
Comment (ganqqwerty) [inline online-services-ui/src/app/shared/components/input-field/input-field.component.html]: This kinda breaks the whole point fo the reactive forms, isn't it?

File: online-services-ui/src/app/shared/components/input-field/input-field.component.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+})
+export class InputFieldComponent implements OnInit {
+
+  @HostBinding('class') readonly class = 'form-group d-block d-md-flex';
+
+  @Input() type = 'text';
+  @Input() label: string;
+  @Input() control: FormControl;
+  @Input() validationMessages?: Array<ValidationMessage>;
+  @Input() min?: number;
```
Comment (ganqqwerty) [inline online-services-ui/src/app/shared/components/input-field/input-field.component.ts]: we don't use min, max, minLength, maxLength attributes in reactive forms, we use Validators instead. Will this code work with validators?

File: online-services-ui/src/app/shared/components/input-field/input-field.component.html (hunk 1 above)
Comment (99999989) [reply online-services-ui/src/app/shared/components/input-field/input-field.component.html]: as discussed, I would keep it as it restricts the input values (what maybe is desired)

File: online-services-ui/src/app/shared/components/input-field/input-field.component.ts (hunk 2 above)
Comment (99999989) [reply online-services-ui/src/app/shared/components/input-field/input-field.component.ts]: as discussed, I would keep it as it restricts the input values (what maybe is desired)

**Pull request merged**